app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,40 @@
+<nav class="bg-white dark:bg-gray-900 fixed w-full z-20 top-0 start-0 border-b border-gray-200 dark:border-gray-600">
+  <div class="max-w-screen-xl flex flex-wrap items-center justify-between mx-auto p-4">
+  <div class="flex items-center space-x-3 rtl:space-x-reverse">
+      <img src="https://www.ajc.com/resizer/U7zTVvkmxcxhAUi-EBgBB0DDokA=/814x458/cloudfront-us-east-1.images.arcpublishing.com/ajc/IMBWXAEJ4HWWVT6BKQAQPFOGAM.jpg" class="h-8 rounded-full" alt="Flowbite Logo">
+      <span class="self-center text-2xl font-semibold whitespace-nowrap dark:text-white">Afrikan Recipe</span>
+  </div>
+  <div class="flex md:order-2 space-x-3 md:space-x-0 rtl:space-x-reverse">
+      <button type="button" class="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">
+        <%= link_to "Log Out", destroy_user_session_path, method: :delete, data: { turbo_method: :delete } %>
+      </button>
```

Comment:
- [ ] The logout functionality has bug to be fixed. Once I click to log out, it throws an error from the user controller and I can see you used user separately and even devise from the view, I will encourage you to remove the user and stick to using devise only as it is the required gem. I had to run the app with different user from incognito. I will encourage you to test it too for you to be double sure.
![image](https://github.com/fatmahussein/Recipe-app/assets/86200356/c36df337-09b7-4f2b-958b-45794205fbee)

Let me know if you not clear.